README.md

```diff
@@ -32,6 +32,19 @@ Add the following line to your Cartfile.
 git "git@github.com:JamitLabs/JamitFoundation.git"
 ```
 
+### CocoaPods
+
+Add the following line to your Podfile.
+
+```swift
+pod 'JamitFoundation', :git => 'https://github.com/knothed/JamitFoundation.git', :tag => '1.4.2', :inhibit_warnings => true
```

Comment:
Why is there `knothed` mentioned here? Is there something missing in the main repository so you needed to fork it?

Reply:
I forgot to change it :D after the MR, of course the main repository should be referenced.